var/spack/repos/builtin/packages/chgcentre/package.py

```diff
@@ -18,3 +18,4 @@ class Chgcentre(CMakePackage):
 
     depends_on("casacore")
     depends_on("gsl")
+    patch('main.patch')
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
Done